src/clients/python/examples/grpc_image_client.py

```diff
@@ -30,6 +30,7 @@
 from PIL import Image
 import os
 import sys
+import struct
 
 import grpc
 from tritongrpcclient import grpc_service_pb2
```

Comment:
the imports can be comma separated such as `from a import x, y`

Reply:
Done